apps/studio/src/components/common/ContextMenu.vue

```diff
@@ -38,6 +39,7 @@
 
 <script lang="ts">
 
+// NOTE (@day): we can't use the store here for some reason
```

Comment:
I think because context menus are dynamically mounted? There's probably a way to do it